suite-common/thp/src/autoInitThpAfterDeviceConnectionThunk.ts

```diff
@@ -0,0 +1,19 @@
+import { createThunk } from '@suite-common/redux-utils';
+import { acquireDevice } from '@suite-common/wallet-core';
+import { Device } from '@trezor/connect';
+
+import { THP_PREFIX } from './thpActions';
+
+type AutoInitThpAfterDeviceConnectionThunkParams = {
+    device: Device;
```

Comment:
This fixes TS.
I see that this thunk is unused even in the final PR #19373 so I surmise this won't cause any other problem :smile: 
```suggestion
    device: TrezorDevice;
```

Reply:
Its here: and it is a mess https://github.com/trezor/trezor-suite/pull/19373/files#diff-2580f7331a6878d62ed30bb9de2c33adbf4566b5d67d8cf33f5705f48af51930R520

We may try together to figure out how to entagle this type magic